web/electron/paths.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ *  Copyright (c) 2021 the Octant contributors. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+import { app } from 'electron';
+import * as path from 'path';
+import * as os from 'os';
+
+export const tmpPath = path.join(os.tmpdir(), 'octant');
+export const apiLogPath = path.join(tmpPath, 'api.out.log');
```

Comment:
I merged the timestamp fix in to master, lets make sure that gets moved over to here.